notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb

```diff
@@ -0,0 +1,1648 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To optimize memory consumption for models with undefined dimensions in run-time, model conversion API provides the capability to define boundaries of dimensions. The boundaries of undefined dimension can be specified with ellipsis. For example, launch model conversion for the ONNX Bert model and specify a boundary for the sequence length dimension:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
+      "[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.\n",
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.xml\n",
+      "[ SUCCESS ] BIN file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/distilbert.bin\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Model Optimizer CLI\n",
+    "\n",
+    "! mo --input_model model/distilbert.onnx --input input_ids,attention_mask --input_shape [1,10..128],[1,10..128] --output_dir model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Python conversion API\n",
+    "from openvino.tools.mo import convert_model\n",
+    "\n",
+    "\n",
+    "ov_model = convert_model(ONNX_NLP_MODEL_PATH, input=[\"input_ids\", \"attention_mask\"], input_shape=[[1, \"10..128\"],[1, \"10..128\"]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Cutting Off Parts of a Model\n",
+    "\n",
+    "The following examples are the situations when model cutting is useful or even required:\n",
+    "\n",
+    "* A model has pre- or post-processing parts that cannot be translated to existing OpenVINO operations.\n",
+    "* A model has a training part that is convenient to be kept in the model but not used during inference.\n",
+    "* A model is too complex be converted at once, because it contains a lot of unsupported operations that cannot be easily implemented as custom layers.\n",
```

Comment:
```suggestion
    "* A model is too complex to be converted at once because it contains many unsupported operations that cannot be easily implemented as custom layers.\n",
```